contentcuration/contentcuration/views/base.py

```diff
@@ -89,11 +89,7 @@ def base(request):
 
 
 def health(request):
-    c = Channel.objects.first()
```

Comment:
This also removes a small information leak, where we give away the first ever channel's name.

Reply:
it's great that we avoid the information leak from the health check now 🎉one thing I'm curious - did we involve channel querying here for checking the health of database? basically, I'm wondering if we need to include the health of database in the health-check api. thank you!

Reply:
Hey! I opted to remove it, since if we depend on the DB being up, that'll prevent us in the future from using a cache entirely to serve user requests, i.e. when the cache is up but the DB is down, the health checks will fail and we can't serve user requests either way.

To allow this functionality, I opted to remove the DB dependency on health checks. Let me know if that's a serious concern!

Reply:
makes sense to me! thank you for the explanation :)